### PR TITLE
Updates made for new user training

### PR DESCRIPTION
**Pull request by ChristinaLK**

### CODE REVIEW

**Comment by ChristinaLK** [comment]
@rachellombardi can you review?

File: README.md [hunk 1]
```diff
@@ -5,21 +5,49 @@ usage varies from book to book or author to author.
...
+
+	$ rm counts.Alices_Adventures_in_Wonderland_by_Lewis_Carroll.tsv
 
+### Create a Submit File
 
-	$ cat wordcount-alice.submit
+To submit a single job that runs this command and analyzes the 
+Alice's Adventures in Wonderland book, we need to translate this command 
+into HTCondor submit file syntax. The two main components we care about 
+are the actual command, and the needed input files.
```
Comment (rachellombardi) [inline README.md]:
```suggestion
into HTCondor submit file syntax. The two main components we care about 
are (1) the actual command and (2) the needed input files. 
```

File: README.md [hunk 2]
```diff
@@ -5,21 +5,49 @@ usage varies from book to book or author to author.
...
+The input file for this job is the `Alices_Adventures_in_Wonderland_by_Lewis_Carroll.txt` 
+text file. We include that in the following submit file option: 
 
 	transfer_input_files    = Alices_Adventures_in_Wonderland_by_Lewis_Carroll.txt
+
+There are other submit file options that control other aspects of the job, like 
+where to save error and logging information, and how many resources to request per 
+job. 
+
+This tutorial has a sample submit file (`wordcount.sub`) with most of this information filled in:
```
Comment (rachellombardi) [inline README.md]:
```suggestion
This tutorial has a sample submit file (`wordcount.sub`) with most of these submit file options filled in: 
```

File: README.md [hunk 3]
```diff
@@ -48,50 +80,57 @@ Now suppose you wanted to analyze multiple books - more than one at a time.
...
-	request_cpus   = 1
-	request_memory = 512MB
-	request_disk   = 512MB
+Create a copy of our existing submit file, that we can use for this job submission. 
+
+	$ cp wordcount.sub wordcount-many.sub
+
+Then, open the file with a text editor and go to the end. We want to tell the 
+`queue` keyword to use our list of inputs to submit jobs. This syntax looks like 
+this:
```
Comment (rachellombardi) [inline README.md]:
```suggestion
`queue` keyword to use our list of inputs to submit jobs. The default syntax looks like this: 

 queue <item> from <list> 
 
 Therefore, when we modify this syntax to fit our example, we get: 
```

File: README.md [hunk 4]
```diff
@@ -48,50 +80,57 @@ Now suppose you wanted to analyze multiple books - more than one at a time.
...
+Therefore, every time we used the name of the book in our submit file (in the previous example, 
+everywhere you see "Alices_Adventures_in_Wonderland_by_Lewis_Carroll.txt") should be 
+replaced with a variable. HTCondor's variable syntax looks like this: `$(variablename)`
 
-	$ ls *.txt > book.list 
+So the following lines in the submit file should be changed to use the variable `$(book)`: 
+
+	arguments = $(book)
+
+	transfer_input_files    = $(book)
```
Comment (rachellombardi) [inline README.md]:
```suggestion
	transfer_input_files = $(book)
```

File: wordcount.sub [hunk 5]
```diff
@@ -1,7 +1,8 @@
-executable = wordcount.py
-arguments = $(book)
+executable = 
+arguments = 
+
+transfer_input_files    =
```
Comment (rachellombardi) [inline wordcount.sub]:
```suggestion
transfer_input_files = 
```

**Review by rachellombardi — COMMENTED** [review]
Looks great!

**Pull request merged**